README.md

```diff
@@ -281,6 +281,12 @@ You can specify multiple values for `ThresholdType` by separating them with comm
 
 You can ignore a method or an entire class from code coverage by creating and applying the `ExcludeFromCodeCoverage` attribute present in the `System.Diagnostics.CodeAnalysis` namespace.
 
+You can also ignore additional attributes by using the `ExcludeByAttribute` property:
+
+```bash
+dotnet test /p:CollectCoverage=true /p:ExcludeByAttribute="ObsoleteAttribute,GeneratedCodeAttribute,CompilerGeneratedAttribute"
+```
```

Comment:
You should mention that the `Attribute` suffix is optional. See my other comment

Reply:
I had a sight fear since it wasn't using the fully qualified name that it could cause issues, but I can make the it try to do the short name resolution like the compiler as well.